plugins/roll.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: roll
+spec:
+  homepage: https://github.com/LifeWay/kubectl-roll-plugin
+  shortDescription: Slowly delete pods by namespace
+  version: v1.0.2
+  description: |
+    A utility to slowly delete pods by namespace with
+    a set amount of seconds between each deleted resource.
```

Comment:
You should mention that this triggers a rolling update of Deployments, ReplicaSets, and StatefulSets. In particular, you need to point out that users must set up readiness probes and use the rolling-update strategy to avoid downtimes.

Btw, waiting between each deleted resource is not doing what it suggests.